modules/cnf-topology-aware-lifecycle-manager-PAO-update.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+// Epic CNF-4767 (4.11), Story TELCODOCS-593
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="talm-PAO-update_{context}"]
+= Removing Performance Addon Operator subscriptions from deployed clusters
+
+In earlier versions of {product-title}, the Performance Addon Operator was used to implement automatic tuning to achieve low latency performance for OpenShift applications. In {product-title} 4.11, these functions are part of the Node Tuning Operator. 
+
+Do not install the Performance Addon Operator on clusters running {product-title} 4.11 or later. When you upgrade to {product-title} 4.11, the Node Tuning Operator automatically removes the Performance Addon Operator. However, you  need to manually remove any Performance Addon Operator subscriptions to prevent a reinstallation of the Operator. The reference DU profile includes the Performance Addon Operator in the `common-ranGen.yaml` `PolicyGenTemplate`. To remove the subscription from deployed spoke clusters, you must update `common-ranGen.yaml`.
+
+.Prerequisites
+
+* Create a Git repository where you manage your custom site configuration data. The repository must be accessible from the hub cluster and be defined as a source repository for Argo CD.
+* Update to {product-title} 4.11. 
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Change the `complianceType` to `mustnothave` for the Performance Addon Operator namespace, Operator group, and subscription in the `common-ranGen.yaml` file.
++
+[source,yaml]
+----
+ -  fileName: PaoSubscriptionNS.yaml
+    policyName: "subscriptions-policy"
+    complianceType: mustnothave
+ -  fileName: PaoSubscriptionOperGroup.yaml
+    policyName: "subscriptions-policy"
+    complianceType: mustnothave
+ -  fileName: PaoSubscription.yaml
+    policyName: "subscriptions-policy"
+    complianceType: mustnothave
+----
++
+. Merge the changes with your custom site repository and wait for the ArgoCD application to synchronize the change to the hub cluster. The state of the common policy changes to `Non-Compliant`.
+. Apply the change to your target clusters using the {cgu-operator-full}. For more information about rolling out configuration changes, see the _Additional resources_ section.
+. Monitor the process. When all clusters show compliance with the updated policy, the Performance Addon Operator has been removed from all clusters. Ensure that the policy is compliant by running the following command:
++
+[source,terminal]
+----
+$ oc get policies --all-namespaces
```

Comment:
It would be better to run 
oc get policy -n <namespaceOfCommonPolicyGenTemplate> common-subscriptions-policy
and show that it is compliant with all clusters. If the status is Compliant all clusters are compliant. If the status is NonCompliant or blank it is not compliant.

Reply:
I updated the code snippet with your suggestion, thanks. I added a bit more detail in the lead-in sentence to this code snippet, per your feedback. Although I didn't mention the nonCompliant status. It might be a bit overkill to detail that and seems a bit obvious: if the status is nonCompliant the policy is not compliant...

Reply:
I think in this case the results are pretty clear but an example output would look like this:

```
$ oc get policy -n ztp-common common-subscriptions-policy
NAME                          REMEDIATION ACTION   COMPLIANCE STATE   AGE
common-subscriptions-policy   inform               Compliant          13d

```